.github/workflows/demo-simple.yaml

```
@@ -0,0 +1,33 @@
+name: Build simple app
+
+on:
+  workflow_dispatch: { }
+  push: { }
+  pull_request: { }
```

Prevent double workflow runs
```suggestion
  push:
    branches: [ main ]
  pull_request:
    branches: [ main ]
```